src/pubsub.js

```diff
@@ -9,8 +9,14 @@
     'use strict';
 
     var PubSub = {};
-    root.PubSub = PubSub;
-    factory(PubSub);
+    /* If PubSub event is passed from another app, then root.PubSub will already be present and we can assign that to PubSub variable.
```

Comment:
This comment can be removed, instead the suggested warning will make it clear why this is done ... as will this pull request :)